modules/homa/src/lib.rs

```diff
@@ -968,11 +956,11 @@ pub mod module {
 			T::Currency::withdraw(T::LiquidCurrencyId::get(), &Self::account_id(), total_redeem_amount)
 		}
 
-		pub fn should_bump_local_current_era(block_number: T::BlockNumber) -> bool {
-			block_number
-				.checked_add(&Self::bump_era_prefix())
-				.and_then(|n| n.checked_rem(&Self::bump_era_frequency()))
-				.map_or(false, |n| n.is_zero())
+		pub fn should_bump_local_current_era(relaychain_block_number: RelayChainBlockNumberOf<T>) -> bool {
+			relaychain_block_number
+				.checked_sub(&Self::last_era_bumped_block())
+				.and_then(|n| n.checked_div(&Self::bump_era_frequency()))
+				.map_or(false, |n| n >= One::one())
```

Comment:
do we need to support bump multiple eras? it will happen when network stalled for more than one era